### PR TITLE
Fix bundler for uniform textures

### PR DESCRIPTION
**Pull request by tallytalwar**
1. any uniform could be a texture, so try all
2. uniform could refer a texture from the scene textures or can
explicitly specify a texture file

### CODE REVIEW

File: tangram_bundler/__init__.py [hunk 1]
```diff
@@ -25,25 +44,17 @@ def addDependencies(file_list, filename):
 
     # Search for textures urls
     if 'textures' in yaml_file:
-        for font in yaml_file['textures']:
```
Comment (nvkelso) [inline tangram_bundler/__init__.py:28]: we still need the fonts thing until it's made optional, no?

Comment (nvkelso) [reply tangram_bundler/__init__.py:28]: Oh, this old code is all matters of awkward.

Comment (tallytalwar) [reply tangram_bundler/__init__.py:28]: Ohh I just renamed `font` to `texture`

**Comment by nvkelso** [comment]
Also bump version in:

- https://github.com/tangrams/bundler/blob/master/setup.py#L6

**Review by matteblair — APPROVED** [review]

**Review by nvkelso — APPROVED** [review]

**Pull request merged**